initramfs-module/Containerfile

```diff
@@ -0,0 +1,8 @@
+FROM quay.io/fedora/fedora-coreos:stable
+RUN KERNEL=$(rpm -q kernel | cut -c 8-); \
+    dracut --reproducible -v --add 'ostree' -f --no-hostonly --omit-drivers 'nouveau' --omit 'nfs' \
```

Comment:
One thing we could note here is that it's now a best practice to ensure that the "base image" has its initramfs arguments embedded in the image.  https://github.com/coreos/fedora-coreos-config/pull/1828

I think at this time, the `--omit nfs` is not needed (per that PR).

Oh I see, it looks like you may have copied these arguments from the RHCOS one, which has these others.

Maybe if we're aiming to handle RHCOS here we could at least link to
https://github.com/openshift/os/blob/1f2c0eb7e370d2412db15fa28556f419ddf73c5d/common.yaml#L37

But ideally in the future we do the same change for RHCOS that we did for FCOS and then this bit isn't necessary.

Reply:
Thanks! I will link to both files in a comment then, that way we provide enough guideline to both RHCOS and FCOS users.

Reply:
I am re reading this, does that mean that for --no-hostonly we don't need to add it either because it's also provided on [overlay.d/05core/usr/lib/dracut/dracut.conf.d/fcos-nohostonly.conf](https://github.com/coreos/fedora-coreos-config/pull/1828/files#diff-bf395571fcbd550f0ca0a32239e1a0e9efd37ace9fa2aa57199d8edb58151f6cR2) ?

Reply:
Yep!